addon/services/liveagent.js

```diff
@@ -1,11 +1,13 @@
-import Service, { inject as service } from '@ember/service';
+import Service from '@ember/service';
 import { getOwner } from '@ember/application';
 
 export default class LiveAgentService extends Service {
-  @service fastboot;
+  isFastBoot = typeof FastBoot !== 'undefined';
```

Comment:
Using `fastboot` injection in applications without `ember-cli-fastboot` throws error...


```
Assertion Failed: Attempting to inject an unknown injection: 'service:fastboot'
```